samples/app-localization/csharp/README.md

```diff
@@ -13,13 +13,20 @@ extensions:
 urlFragment: officedev-microsoft-teams-samples-app-localization-csharp
 ---
 
-# Teams App Localization
+# Teams App Localization [[Try It Yourself](/samples/app-localization/csharp/Try-It-Yourself.md)]
```

Comment:
Adding this to title will impact the display name in few [other galleries](https://learn.microsoft.com/en-us/samples/browse/) as they use H1 heading (first #) as title. Let's keep the separate section. Also, should we move these manifest files to their respective folders?

Reply:
Removed it.